pkg/controller/cassandracluster/generator.go

```diff
@@ -630,6 +591,55 @@ func bootstrapContainerEnvVar(cc *api.CassandraCluster, status *api.CassandraClu
 			},
 		},
 	}
+	commonEnvVars := commonBootstrapCassandraEnvVar(cc)
+	bootstrapEnvVars = append(bootstrapEnvVars, commonEnvVars...)
+	return bootstrapEnvVars
+}
+
+func cassandraContainerEnvVar(cc *api.CassandraCluster) []v1.EnvVar {
```

Comment:
You can get rid of that function and just call `commonBootstrapCassandraEnvVar(cc)` at https://github.com/Orange-OpenSource/casskop/pull/273/files#diff-8f39a9e34ee5c81ededc0c74d2d2e8ffc5dece79eab1cd4223461aa073b3a9c4R771 like you did above

Reply:
It's probably premature, but I was trying to make it clear which variables get defined where: cassandra-only env vars get defined in `cassandraContainerEnvVar()`, bootstrap-only env vars get defined `bootstrapContainerEnvVar()`, and the common ones go in a common function.

Since the cassandra container ONLY contains common variables right now, it's not really necessary, but I was trying to make it obvious where to define cassandra-only env vars in the future.

I'll scratch that idea and just call `commonBootstrapCassandraEnvVar()` directly.